### ISSUE TITLE
An apt-get install of bfgminer on Ubuntu 10.04 x86 fails against libssl1.0.0

### ISSUE THREAD

**Issue by dwangoac** [luke-jr/bfgminer#94]
Steps:
Add ppa:unit3/bfgminer to an Ubuntu 10.04 x86 (32-bit) system and attempt to install the bfgminer package.

Result:
The installation will fail with:

```
$ sudo apt-get install bfgminer
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
  bfgminer: Depends: libssl1.0.0 but it is not installable
E: Broken packages
```

Expected:
Either libssl should not be required or an appropriate 10.04 dependency should exist.

Additional information:
I have not yet tested this on my Ubuntu 12.04 x86 system but I can do that as needed.

**Comment by luke-jr** [comment]
@unit3 Why is libssl a dependency at all? O.o;

**Comment by nergdron** [comment]
Not sure, it was listed as a dep in the packaging code I was working from, and binaries I built seemed to be linked to it, so I assumed it was necessary. I'm also uncertain how the 10.04 package built if that libssl package isn't available on 10.04.

I'll look at stripping out the dep tomorrow, since I've gotta get builds for the new release out.

**Comment by nergdron** [comment]
@dwangoac I've uploaded new 2.6.4 packages that don't have this dependency. Can you test on this system and let me know if they install and work ok?

Thanks!

**Comment by dwangoac** [comment]
Confirmed fixed - thank you for the quick fix!  I performed an apt-get update and an apt-get install bfgminer which installed bfgminer 2.6.4-0lucid3 with no problems.  I was able to launch bfgminer with the same options that I used in cgminer and I was immediately able to connect to Eligius.

**Comment by dwangoac** [comment]
This appears to have been reintroduced with version 2.9.1-0lucid1:

$ apt-cache showpkg bfgminer
Package: bfgminer
Versions: 
2.9.1-0lucid1 (/var/lib/apt/lists/ppa.launchpad.net_unit3_bfgminer_ubuntu_dists_lucid_main_binary-i386_Packages)
 Description Language: 
                 File: /var/lib/apt/lists/ppa.launchpad.net_unit3_bfgminer_ubuntu_dists_lucid_main_binary-i386_Packages
                  MD5: 94994dc9559f3c56aebd34cc6c037006

2.6.4-0lucid3 (/var/lib/dpkg/status)
 Description Language: 
                 File: /var/lib/dpkg/status
                  MD5: 94994dc9559f3c56aebd34cc6c037006

Reverse Depends: 
Dependencies: 
2.9.1-0lucid1 - libjansson4 (0 (null)) libssl1.0.0 (0 (null)) libudev0 (0 (null)) libcurl3 (0 (null)) libncurses5 (0 (null)) libusb-1.0-0 (0 (null)) 
2.6.4-0lucid3 - libudev0 (0 (null)) libcurl3 (0 (null)) libncurses5 (0 (null)) libusb-1.0-0 (0 (null)) 
Provides: 
2.9.1-0lucid1 - 
2.6.4-0lucid3 - 
Reverse Provides: 

$ sudo apt-get install bfgminer
Reading package lists... Done
Building dependency tree  
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
  bfgminer: Depends: libssl1.0.0 but it is not installable
E: Broken packages

Can you provide the same fix?  Until then I'm stuck on 2.6.4-0lucid3.  Thanks!

**Comment by luke-jr** [comment]
@dwangoac Why reopen?

**Comment by dwangoac** [comment]
As a workaround, I've discovered that while it is not possible to force apt-get to ignore a missing dependency it _IS_ possible to get dpkg to ignore them with --force-all.  As a temporary workaround, I downloaded https://launchpad.net/~unit3/+archive/bfgminer/+files/bfgminer_2.9.1-0lucid1_i386.deb directly and issued the command dpkg --force-all -i  bfgminer_2.9.1-0lucid1_i386.deb which installed the package.  Ignoring the missing dependency does not appear to have caused any problems and I'm up and mining.  Just a heads-up for anyone else who encounters this in the meantime. :)
